<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MSWin</class>
 <widget class="QMainWindow" name="MSWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1246</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Princeton LC-MS/MS Data Viewer and Analyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>0</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Select</string>
       </attribute>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Conditions</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="listCond"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Replicate sets</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QListWidget" name="listRep">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Instrument runs</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QListWidget" name="listIrun"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Info</string>
       </attribute>
       <widget class="QLabel" name="rtLabel">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>35</y>
          <width>226</width>
          <height>56</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>32383.382</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>15</y>
          <width>142</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Retention time (s):</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>m/z (Daltons):</string>
        </property>
       </widget>
       <widget class="QLabel" name="mzLabel">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>125</y>
          <width>253</width>
          <height>44</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>600.38287382</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabMainDisplay">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>MS1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGraphicsView" name="graphicsView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>640</width>
            <height>480</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Isotope</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTreeWidget" name="isotopeTree">
          <column>
           <property name="text">
            <string>Protein Group</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>log2 H/L</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>log2 M/L</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>log2 H/M</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>XIC</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTreeWidget" name="xicTree">
          <column>
           <property name="text">
            <string>Protein Groups</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1246</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Recalibrate"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Label_Free">
    <property name="title">
     <string>&amp;Label Free</string>
    </property>
    <addaction name="actionSave_TXT"/>
    <addaction name="actionSave_Align_XICs_TXT"/>
   </widget>
   <widget class="QMenu" name="menu_SILAC">
    <property name="title">
     <string>&amp;Isotope Pairs</string>
    </property>
    <addaction name="actionSave_SILAC_TXT"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Label_Free"/>
   <addaction name="menu_SILAC"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionSave_TXT">
   <property name="text">
    <string>Save Tab Delimited...</string>
   </property>
  </action>
  <action name="actionSave_SILAC_TXT">
   <property name="text">
    <string>Save Tab Delimited</string>
   </property>
  </action>
  <action name="actionSave_Align_XICs_TXT">
   <property name="text">
    <string>Save Align XICs Tab Delimited...</string>
   </property>
  </action>
  <action name="action_Recalibrate">
   <property name="text">
    <string>&amp;Recalibrate...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
